test/helpers/factory.js

```diff
@@ -18,11 +19,11 @@ export function gistData({html, css, javascript, enabledLibraries = []} = {}) {
       content: javascript,
     });
   }
-  if (!isEmpty(enabledLibraries)) {
```

Comment:
You know what—I’m the one who’s not making sense. Somehow I failed to notice this was in the test helper code. What you’ve got here is entirely reasonable!